tool/src/org/antlr/v4/codegen/target/GoTarget.java

```diff
@@ -54,6 +57,26 @@ protected void addBadWords() {
 	}
 
 	@Override
+	protected void genFile(Grammar g, ST outputFileST, String fileName) {
+		super.genFile(g, outputFileST, fileName);
+		gofmt(fileName); // consider commenting out when debugging
+	}
+
+	private void gofmt(String fileName) {
+		// If the generated source file is for any reason invalid, gofmt will not change the file and generate output on
+		// stderr. The output of stderr will be lost, gofmt may block asynchronously if the output buffer is not
```

Comment:
I don't believe gofmt has output with those flags. Is that still an issue? If so, perhaps the right thing to do here is to read from its outputs and wait until the subprocess exits.